draft-ietf-taps-interface.md

```diff
@@ -1671,22 +1671,6 @@ This enumerated property specifies the application's preferred tradeoffs for
 sending this Message; it is a per-Message override of the Capacity Profile
 protocol and path selection property (see {{prop-cap-profile}}).
 
```

Comment:
protocol and path selection --> connection (as this property is a connection property)

Reply:
Fixing! I don't know how this "protocol and" slipped in there - good catch, thanks!